src/Persistence/Templates/WellKnownTemplateIds.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Templates;
+
+public static class WellKnownTemplateIds
```

Comment:
@petrochuk  has added something similar in https://github.com/microsoft/PowerApps-Tooling/pull/636/files#diff-ac4572e52bc9a7f059eddb37ee233946299b268991310aaf4caefd7c1484ad22 . I'd wait for his PR to go in first and then adapt the changes.

Reply:
I think it makes more sense to have names and ids in one place instead of having two separate classes: one for names and one for ids

Reply:
This class only hods IDs. the names are extra metadata which are out of scope of document template Ids.
I've remerged with master and used these constants in your 'BuiltInTemplates' class so there's still a single source of truth.

Reply:
I think it would be useful to have ids in the same place as names